tests/storages_tests/rdb_tests/test_storage.py

```diff
@@ -399,3 +400,34 @@ def failed_trial_callback(study: Study, trial: FrozenTrial) -> None:
             with patch.object(storage, "failed_trial_callback", wraps=failed_trial_callback) as m:
                 study.optimize(lambda _: 1.0, n_trials=1)
                 m.assert_called_once()
+
+
+def test_RetryFailedTrialCallback() -> None:
```

Comment:
A cosmetic but let's stick to the snake case convention.
```suggestion
def test_retry_failed_trial_callback() -> None:
```